src/redux/store/store.js

```diff
@@ -0,0 +1,12 @@
+import { configureStore } from "@reduxjs/toolkit";
+import folderReducer from "../slices/folderSlices";
+import logger from "redux-logger";
+
+const store = configureStore({
+  reducer: {
+    folder: folderReducer,
+  },
+  middleware: (getDefaultMiddleware) => getDefaultMiddleware().concat(logger),
+});
+
+export default store;
```

Comment:
```suggestion
export default store;

```
마지막 줄 개행이 있어야할 것 같습니다!